src/theme/all.css

```diff
@@ -605,3 +605,58 @@ div.olMap div.gx-overlay-playback {
 .thumb-mapbox-world-print {
     background-image:url('img/thumbs/mapbox/world-print.png');
 }
+.x-form-spinner-proxy{
+    /*background-color:#ff00cc;*/
+}
+.x-form-field-wrap .x-form-spinner-trigger {
+    background:transparent url('img/spinner.gif') no-repeat 0 0;
+    right: 0px;
+}
+.x-form-field-wrap .x-form-spinner-overup{
```

Comment:
I'd leave it up to the applications to include the spinner css in their builds instead of duplicating it here.